src/systemsSerology.jl

```diff
@@ -50,10 +50,58 @@ function antigenTables(s::String)
     dfF = CSV.read(joinpath(dataDir, "alter-MSB", "data-function.csv"))
     newdfF = dfF[dfF[!, :ADCC] .!= "NA", :] #ignore subjects who have no ADCC data
     ADCConly = newdfF[:, [:Column1, :ADCC]] #only want ADCC data
+    ADCConly.ADCC = tryparse.(Float64, ADCConly.ADCC) #ADCC data should be Floats not strings
     ADCC = rename!(ADCConly, [:Subject, :ADCC]) #Subjects were called "Column1" before
 
     #join ADCC data with antigen receptor data
     allAntigen = join(rec, ADCC, on = :Subject, kind = :inner)
 
     return allAntigen
 end
+
+""" Assemble the Cube """
+function createCube()
+    dataDir = joinpath(dirname(pathof(FcgR)), "..", "data")
+    dfMA = CSV.read(joinpath(dataDir, "alter-MSB", "meta-antigens.csv"))
+    dfMS = CSV.read(joinpath(dataDir, "alter-MSB", "meta-subjects.csv"))
+    dfMD = CSV.read(joinpath(dataDir, "alter-MSB", "meta-detections.csv"))
+    dfMD.detection = replace.(dfMD.detection, "." => "_")
+    dfMD.Number = 1:22 #Add a column of index numbers that correspond to each receptor/detection
+
+    Cube = Array{Union{Nothing, Float64}}(nothing, 182, 23, 41) #create a the Cube, filled with nothing
+
+    #Subject IDs in column 1 down the cube
+    #Detection number across top of cube - like column names for each slice
+    for i = 2:182
```

Comment:
The ID's and detection numbers should be kept in separate variables.

Reply:
I think I understand what you meant by this, let me know if that's what you wanted in the commit I just pushed 'separate variables.' This loop is a little tricky because I am starting at (1,2) and (2,1) to essentially give the array column and row names starting at (1) in each array of detections and IDS.

Reply:
What I meant is that the subject ID's and DNums *shouldn't* be in the cube. The cube should only be made up of data values.

Reply:
Oh okay. Let me think about this and work on it later, because right now I'm using those as indexes to find the right locations in the cube. Perhaps I can find a way to delete them after the cube is assembled...

Reply:
You can keep them in a separate vector. Their indices will still be the same.